x-pack/plugins/index_lifecycle_management/public/sections/policy_table/components/policy_table/policy_table.js

```diff
@@ -0,0 +1,389 @@
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      selectedPoliciesMap: {},
+      renderConfirmModal: null,
+    };
+  }
+  componentDidMount() {
+    this.props.fetchPolicies(true);
+  }
+  renderEmpty() {
+    return (
+      <EuiEmptyPrompt
+        iconType="managementApp"
+        title={
+          <h1>
+            <FormattedMessage
+              id="xpack.indexLifecycleMgmt.policyTable.emptyPromptTitle"
+              defaultMessage="Create your first index lifecycle policy"
+            />
+          </h1>
+        }
+        body={
+          <Fragment>
+            <p>
+              <FormattedMessage
+                id="xpack.indexLifecycleMgmt.policyTable.emptyPromptDescription"
+                defaultMessage="Index lifecycle policies help you control your index lifecycle"
+              />
+            </p>
+          </Fragment>
+        }
+        actions={this.renderCreatePolicyButton()}
+      />
+    );
+  }
+  renderDeleteConfirmModal = () => {
+    const { policyToDelete } = this.state;
+    if (!policyToDelete) {
+      return null;
+    }
+    return (
+      <ConfirmDelete
+        policyToDelete={policyToDelete}
+        callback={this.handleDelete}
+        onCancel={() => this.setState({ renderDeleteConfirmModal: null, policyToDelete: null })}
+      />
+    );
+  }
+  handleDelete = () => {
+    this.props.fetchPolicies(true);
+    this.setState({ renderDeleteConfirmModal: null, policyToDelete: null });
+  };
+  onSort = column => {
+    const { sortField, isSortAscending, policySortChanged } = this.props;
+    const newIsSortAscending = sortField === column ? !isSortAscending : true;
+    policySortChanged(column, newIsSortAscending);
+  };
+
+  buildHeader() {
+    const { sortField, isSortAscending } = this.props;
+    const headers = Object.entries(HEADERS).map(([fieldName, label]) => {
+      const isSorted = sortField === fieldName;
+      return (
+        <EuiTableHeaderCell
+          key={fieldName}
+          onSort={() => this.onSort(fieldName)}
+          isSorted={isSorted}
+          isSortAscending={isSortAscending}
+          data-test-subj={`policyTableHeaderCell-${fieldName}`}
+          className={'policyTable__header--' + fieldName}
+        >
+          {label}
+        </EuiTableHeaderCell>
+      );
+    });
+    headers.push(
+      <EuiTableHeaderCell
+        key="deleteHeader"
+        data-test-subj="policyTableHeaderCell-delete"
+        style={{ width: 100 }}
+      />
+    );
+    return headers;
+  }
+
+  buildRowCell(fieldName, value) {
+    if (fieldName === 'name') {
+      return (
+        <EuiLink
+          className="policyTable__link"
+          data-test-subj="policyTablePolicyNameLink"
+          href={getPolicyPath(value)}
+        >
+          {value}
+        </EuiLink>
+      );
+    } else if (fieldName === 'coveredIndices' && value) {
+      return (
+        <EuiText>
+          <b>{value.length}</b>{' '}
```

Comment:
also suggest to remove `<b>` and the trailing space

```suggestion
          {value ? value.length : '0'}
```